app/controllers/api/databases.php

```diff
@@ -1673,12 +1676,77 @@ function updateAttribute(
             throw new Exception(Exception::COLLECTION_NOT_FOUND);
         }
 
-        $attributes = $collection->getAttribute('attributes');
+        $queriesFromRequest = Query::parseQueries($queries);
```

Comment:
Let's override `$queries` with the parsed value

Reply:
fixed this